### DISCUSSION TITLE
TIme periods to select different setpoints

### DISCUSSION THREAD

**Discussion by paulashby1966** [jmcollin78/versatile_thermostat#205] [Ideas]
Currently I use the thermostat with just motion detection so when the room is occupied its set to 20 Dec C and when unoccupied 18 Deg C.
It would be nice to also have time periods for day/night periods so at night even if someone gets up for the toilet the heaters will not be trying to reach 20 Deg C.  Having different Eco and comfort set points for day and night would help with this.

**Comment by wiktor-z** [comment]
I recall using this https://github.com/nielsfaber/scheduler-card to configure parameters and values based on time. It might be useful for you as well.

**Comment by jmcollin78** [comment]
Yes the Scheduler is your friend. You can change the preset on a time based. Set the preset to Eco at night and to Activity during the day.

The other thing is that there is delay between motion detection and temperatrure change. You can tune this to avoid switching when the motion is too short.
